polyfills/String/prototype/padEnd/polyfill.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+	Object.defineProperty(String.prototype, 'padEnd', {
+		configurable: true,
+		enumerable: false,
+		value: function padEnd (targetLength, padString) {
+			targetLength = targetLength | 0;
```

Comment:
Using bitwise OR here to floor `targetLength` could cause issues with particularly large numbers. E.g.

```js
3000000000.1 | 0 === -1294967296
```

In the case above the resulting string reaches the maximum length in Chrome Canary and Firefox stable before this becomes an issue. However as [maximum string length can differ between different JavaScript engines](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Resulting_string_too_large) this could feasibly cause problems in the future.

Not sure whether it's a pressing issue but worth documenting. An alternative could be to use:

```js
targetLength = Math.floor(parseInt(targetLength, 10)) || 0;
```